assist/README.md

```diff
@@ -1,8 +1,141 @@
 RACK ASSIST
 ===========
 
+# Introducing RACK ASSIST
+
 (Automated Software Structure Ingestion and System Testing)
 
+The ASSIST toolkit contains several tools that can be used with RACK.
+
+ * ASSIST-DC  -- Data Collection
+ * ASSIST-DI  -- Data Ingestion
+ * ASSIST-DV  -- Data Verification
+ * ASSIST-SADL  -- SADL translation
+ * ASSIST-RACKLIB -- RACK library
```

Comment:
Is there a reason for the different spacing here and in the section headers?

Reply:
Fixed.